### DISCUSSION TITLE
Cannot Sign In

### DISCUSSION THREAD

**Discussion by debwini** [iterate-ch/cyberduck#14968] [Q&A]
I am not able to sign into my Cyberduck account.  I have never had this problem before.

Please advise.
Thank you.

**Comment by tatze96** [comment]
Please elaborate more on which account you are referring to. Additionally, please add a screenshot of the error message, if there are any. Make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

Note: If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider.
